Missed updating a variable name:
```
Interpolating increment for f003
interp_inc 3 OrderedDict([('setup', {'lon_out': 384, 'lat_out': 192, 'lev': 127, 'infile': "'siginc.nc.03'", 'outfile': "'inc.fullres.03'"})])
Traceback (most recent call last):
  File "/lfs/h2/emc/global/noscrub/globalworkflow.ci/GFS_CI_ROOT/PR/2645/global-workflow/ush/calcanl_gfs.py", line 366, in <module>
    calcanl_gfs(DoIAU, l4DEnsVar, Write4Danl, ComOut, APrefix,
  File "/lfs/h2/emc/global/noscrub/globalworkflow.ci/GFS_CI_ROOT/PR/2645/global-workflow/ush/calcanl_gfs.py", line 256, in calcanl_gfs
    job = subprocess.Popen(ExecCMDMPI10_host + ' ' + CalcAnlDir + '/chgres_inc.x', shell=True, cwd=CalcAnlDir)
NameError: name 'ExecCMDMPI10_host' is not defined
+ exglobal_atmos_analysis_calc.sh[1]: postamble exglobal_atmos_analysis_calc.sh 1717275805 1
```

--- a/ush/calcanl_gfs.py
+++ b/ush/calcanl_gfs.py
@@ -253,8 +253,8 @@ def calcanl_gfs(DoIAU, l4DEnsVar, Write4Danl, ComOut, APrefix,
             if launcher == 'srun':
                 os.environ['SLURM_HOSTFILE'] = CalcAnlDir + '/hosts'
             print('interp_inc', fh, namelist)
-            job = subprocess.Popen(ExecCMDMPI10_host + ' ' + CalcAnlDir + '/chgres_inc.x', shell=True, cwd=CalcAnlDir)
-            print(ExecCMDMPI10_host + ' ' + CalcAnlDir + '/chgres_inc.x submitted on ' + hosts[ihost])
+            job = subprocess.Popen(ExecCMDMPI13_host + ' ' + CalcAnlDir + '/chgres_inc.x', shell=True, cwd=CalcAnlDir)
+            print(ExecCMDMPI13_host + ' ' + CalcAnlDir + '/chgres_inc.x submitted on ' + hosts[ihost])
             sys.stdout.flush()
             ec = job.wait()
             if ec != 0:

--- a/ush/calcanl_gfs.py
+++ b/ush/calcanl_gfs.py
@@ -248,7 +248,7 @@ def calcanl_gfs(DoIAU, l4DEnsVar, Write4Danl, ComOut, APrefix,
                         ihost += 1
                         for a in range(0, 5):
                             hostfile.write(hosts[ihost] + '\n')
-                    for a in range(0, 12):  # need 9 more of the same host for the 10 tasks for chgres_inc
+                    for a in range(0, 12):  # need 12 more of the same host for the 13 tasks for chgres_inc
                         hostfile.write(hosts[ihost] + '\n')
             if launcher == 'srun':
                 os.environ['SLURM_HOSTFILE'] = CalcAnlDir + '/hosts'

